### PR TITLE
Add external secret for Akeyless tests

### PR DESCRIPTION
**Pull request by d-goro**
PR adds external secret in order to use it in tests adding Akeyless provider for CSI driver

### CODE REVIEW

File: config/prow/cluster/build/build_kubernetes-external-secrets_customresource.yaml [hunk 1]
```diff
@@ -3,6 +3,19 @@
 ---
 apiVersion: kubernetes-client.io/v1
 kind: ExternalSecret
+metadata:
+  name: akeyless-test-cred
+  namespace: test-pods
+spec:
+  backendType: gcpSecretsManager
+  projectId: kubernetes-upstream
```
Comment (aramase) [inline config/prow/cluster/build/build_kubernetes-external-secrets_customresource.yaml:11]: This `projectId` would need to reference your project. `kubernetes-upstream` is what we use for azure and it's not shared.

Comment (d-goro) [reply config/prow/cluster/build/build_kubernetes-external-secrets_customresource.yaml]: done

**Pull request closed without merging**